subsys/bluetooth/host/Kconfig

```diff
@@ -991,36 +991,21 @@ config BT_DF_VS_CONN_IQ_REPORT_16_BITS_IQ_SAMPLES
 	  format.
 
 endif # BT_DF
-endif # BT_HCI_HOST
 
 config BT_ECC
-	bool "ECDH key generation support"
-	default y if BT_SMP && !BT_SMP_OOB_LEGACY_PAIR_ONLY
-	help
-	  This option adds support for ECDH HCI commands.
-
-config BT_SEND_ECC_EMULATION
-	bool "Emulate ECDH in the Host using PSA Crypto API library"
```

Comment:
`BT_SEND_ECC_EMULATION` was added prior to the 4.0 release, which means that we have a release with this Kconfig, and it was designated as experimental. Don't we need to go through the deprecation process for this?

Reply:
Could we modify `BT_SEND_ECC_EMULATION` to just `select DEPRECATED` and nothing else? 

That way it continues to have the same behavior (as it is enabled by default now), while still providing information to the users that the Kconfig is now deprecated?

Reply:
> BT_SEND_ECC_EMULATION was added prior to the 4.0 release

I just checked, and I can't find any mention of an option with that name in 4.0, so I think I can just omit this part from the migration guide.